src/vmm/src/guest_config/templates_serde.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
why can't this file be placed under `templates` directory?

Reply:
My idea it to have following PR where I remove most things from `template` dir into respective `x86`/`aarch64` modules. Then `guest_config` will only have `x86_64`, `aarch64`, `templates.rs`, `templates_serde.rs` and `mod.rs`